.github/README.md

```diff
@@ -44,7 +42,7 @@ This will configure and compile perl for your platform, run the regression
 tests, and install perl in a subdirectory "`localperl`" of your home directory.
 
 If you run into any trouble whatsoever or you need to install a customized
-version of Perl, you should read the detailed instructions in the "[INSTALL](INSTALL)"
+version of Perl, you should read the detailed instructions in the "[INSTALL](https://github.com/rwp0/perl5/blob/blead/INSTALL)"
```

Comment:
You're linking to https://github.com/**rwp0**/perl5/blob/blead - whoops!

I think it would be better to take this commit and correct it with `git commit --amend` instead of pushing another fixup commit after it, as keeping two commits would just clutter the history.

(ie then update the branch with `git push --force-with-lease`)

(I wonder - can you use relative links? ie would ../INSTALL work here? Is the problem that this file is now down in `.github`? Relative links would be cleaner if they work correctly)

Reply:
Oops, right. Okay, I'll do so. Yes, sadly, the relative links I put before were pointing from `.github` directory.

Reply:
I'm not sure if I could use `git commit --amend` correctly, but changes are committed already.